### PR TITLE
OCPBUGSM-17615 Installation hang on Finalizing

### PR DESCRIPTION
**Pull request by ori-amizur**
/cc @eranco74 
/cc @ronniel1

### CODE REVIEW

File: internal/network/cidr_validations.go [hunk 1]
```diff
@@ -48,6 +48,28 @@ func VerifySubnetCIDR(cidrStr string) error {
...
+	}
+	return y
+}
+
+func VerifyClusterCidrSize(hostNetworkPrefix int, clusterNetworkCIDR string, numberOfHosts int) error {
+	_, cidr, err := net.ParseCIDR(clusterNetworkCIDR)
+	if err != nil {
+		return err
+	}
+	ones, _ := cidr.Mask.Size()
```
Comment (eranco74) [inline internal/network/cidr_validations.go:63]: Consider renaming `ones` to `clusterNetworkPrefix`

**Review by eranco74 — COMMENTED** [review]
Looks good

File: internal/network/cidr_validations.go (hunk 1 above)
Comment (ori-amizur) [reply internal/network/cidr_validations.go:63]: OK

**Review by eranco74 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**